requirements.txt

```diff
@@ -1,2 +1,3 @@
 ckanapi
 ckantoolkit>=0.0.2
+pytz==2012j
```

Comment:
I am just following what is in CKAN so that they stay synced

https://github.com/ckan/ckan/blob/master/requirements.txt#L31